lib/reek/rake/task.rb

```diff
@@ -85,7 +85,9 @@ def initialize(name = :reek)
 
       # @public
       def source_files=(files)
-        raise ArgumentError, no_string_given_for_file_list_warning unless files.is_a?(String)
+        unless files.is_a?(String) || files.is_a?(FileList)
+          raise ArgumentError, no_string_or_filelist_given_for_file_list_warning
```

Comment:
Let's rather drop this ugly method (I know it was there before, but still) and just use:

```Ruby
raise ArgumentError, "File list should be a FileList or a String that can contain"\
  "a glob pattern, e.g. '{app,lib,spec}/**/*.rb'"
```